tests/testthat/test-module_teal.R

```diff
@@ -331,13 +334,13 @@ testthat::describe("srv_teal teal_modules", {
       ),
       expr = {
         testthat::expect_null(modules_output$module_1())
-        session$setInputs(`teal_modules-active_tab` = "module_1")
-        testthat::expect_null(modules_output$module_1())
+        session$setInputs(`teal_modules-active_tab` = "module_2")
+        testthat::expect_identical(modules_output$module_1(), 101L)
```

Comment:
This one is weird, as is the only one where modules return NULL before anything is selected